<?xml version="1.0" encoding="UTF-8"?><record_update table="sn_pipeline_environment">
    <sn_pipeline_environment action="INSERT_OR_UPDATE">
        <instance_credential display_value="Fake_App_Engine_Studio_Credential">e9580dc11b5cf4101e7a6352604bcbbf</instance_credential>
        <instance_id>Validation Failed: Failure code:401</instance_id>
        <instance_type>prod</instance_type>
        <instance_url>https://dev60234.service-now.com/</instance_url>
        <is_controller>false</is_controller>
        <name>Prod Instance</name>
        <sys_class_name>sn_pipeline_environment</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2022-06-22 22:31:13</sys_created_on>
        <sys_id>f69aa6bc2fd01110e775f7ecf699b683</sys_id>
        <sys_mod_count>1</sys_mod_count>
        <sys_name>Prod Instance</sys_name>
        <sys_package display_value="DQ Exception Tracker" source="x_272800_dq_exce_0">346852702f501110e775f7ecf699b681</sys_package>
        <sys_policy/>
        <sys_scope display_value="DQ Exception Tracker">346852702f501110e775f7ecf699b681</sys_scope>
        <sys_update_name>sn_pipeline_environment_f69aa6bc2fd01110e775f7ecf699b683</sys_update_name>
        <sys_updated_by>system</sys_updated_by>
        <sys_updated_on>2022-06-23 03:00:30</sys_updated_on>
    </sn_pipeline_environment>
</record_update>
